readthedocs/projects/models.py

```diff
@@ -1200,7 +1232,7 @@ class HTMLFile(ImportedFile):
     class Meta:
         proxy = True
 
-    objects = HTMLFileManager()
+    objects = HTMLFileManager.from_queryset(HTMLFileQuerySet)()
```

Comment:
We can use the Mixin that I'm proposing here as well and do not use `.from_queryset` avoiding creating another class very similar, I believe.